app/assets/javascripts/discourse/app/widgets/user-notifications-large.js

```diff
@@ -30,9 +30,8 @@ createWidget("large-notification-item", {
 
 export default createWidget("user-notifications-large", {
   html(attrs) {
-    const notifications = attrs.notifications;
```

Comment:
I don't think the changes in this file are worth comitting.

Reply:
@Ahmedgagan just bumping this comment ^^

Reply:
Yes, i already changed this in my last commit.